Cuckoo tests fail on ARM processor due to debug format string

**Environment:**

Jetson Nano board (`/proc/cpuinfo`: ARMv8 Processor rev 1 (v8l))
RedisBloom 2.2.3 (but issue also happens with older versions)

**Observation**

Building RedisBloom and running tests fails
```
$> make
[... build works without problems ...]
$> make test
```
breaks with
```
======================================================================
FAIL: test_bucket_size (__main__.CuckooTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "cuckoo.py", line 227, in test_bucket_size
    self.assertEqual(self.cmd('CF.DEBUG a'), 'bktsize:1 buckets:64 items:1000 deletes:0 filters:18 max_iterations:20 expansion:1')
AssertionError: 'bktsize:1 buckets:64 items:1000 deletes:0 filters:18 max_iterations:20 expansion:545460846593' != 'bktsize:1 buckets:64 items:1000 deletes:0 filters:18 max_iterations:20 expansion:1'

======================================================================
FAIL: test_expansion (__main__.CuckooTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "cuckoo.py", line 248, in test_expansion
    self.assertEqual(self.cmd('CF.DEBUG a'), 'bktsize:2 buckets:32 items:1000 deletes:0 filters:17 max_iterations:20 expansion:1')
AssertionError: 'bktsize:2 buckets:32 items:1000 deletes:0 filters:17 max_iterations:20 expansion:545460846593' != 'bktsize:2 buckets:32 items:1000 deletes:0 filters:17 max_iterations:20 expansion:1'

======================================================================
FAIL: test_insert (__main__.CuckooTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "cuckoo.py", line 135, in test_insert
    self.assertEqual('bktsize:2 buckets:8192 items:1 deletes:0 filters:1 max_iterations:20 expansion:1', d3.decode())
AssertionError: 'bktsize:2 buckets:8192 items:1 deletes:0 filters:1 max_iterations:20 expansion:1' != u'bktsize:2 buckets:8192 items:1 deletes:0 filters:1 max_iterations:20 expansion:545460846593'

======================================================================
FAIL: test_max_iterations (__main__.CuckooTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "cuckoo.py", line 171, in test_max_iterations
    self.assertEqual('bktsize:2 buckets:8 items:0 deletes:0 filters:1 max_iterations:10 expansion:1', d1)
AssertionError: 'bktsize:2 buckets:8 items:0 deletes:0 filters:1 max_iterations:10 expansion:1' != 'bktsize:2 buckets:8 items:0 deletes:0 filters:1 max_iterations:10 expansion:545460846593'
```

**I found this fix:**

Change format string in `rebloom.c` at line 965 to:

```
        ctx, "bktsize:%lu buckets:%lu items:%lu deletes:%lu filters:%lu max_iterations:%lu expansion:%u",
```
(from originally `expansion: %lu`)

and all tests work.

So the test fail looks like to be only a formatting problem, that somehow occurs on ARM processors due to some different byte sizes.

I just checked that building & testing RedisBloom on a non-ARM processor (Intel i7-7700HQ) still works when changing the format string for expansion to `expansion:%u`.

--- a/src/rebloom.c
+++ b/src/rebloom.c
@@ -960,7 +960,7 @@ static int CFDebug_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, i
     }
 
     RedisModuleString *resp = RedisModule_CreateStringPrintf(
-        ctx, "bktsize:%lu buckets:%lu items:%lu deletes:%lu filters:%lu max_iterations:%lu expansion:%lu",
+        ctx, "bktsize:%u buckets:%lu items:%lu deletes:%lu filters:%u max_iterations:%u expansion:%u",
         cf->bucketSize, cf->numBuckets, cf->numItems, cf->numDeletes, 
         cf->numFilters, cf->maxIterations, cf->expansion);
     return RedisModule_ReplyWithString(ctx, resp);

